frontend/public/module/k8s/label-selector.js

```diff
@@ -0,0 +1,257 @@
+/* eslint-disable */
+
+// TODO (Jon) Fix lint issues in this file
```

Comment:
Probably worth opening a tech debt issue to track.

Reply:
Yeah, I started looking into this but it turned into something bigger than just some style changes. Some of the lint errors relate to variable declarations in switch statements, which will require some refactoring that I didn't feel was in the scope of this story.

Reply:
We could let `eslint --fix` fix what it can, and then exclude a couple of the tricky errors for now (with a tech debt story to fix)